terralab/utils.py

```diff
@@ -15,11 +15,14 @@
 
 from teaspoons_client import ApiException
 
+from terralab.constants import SUPPORT_EMAIL
 from terralab.log import add_blankline_after
 
 
 LOGGER = logging.getLogger(__name__)
 
+SUPPORT_EMAIL_TEXT = f"For help troubleshooting, email {SUPPORT_EMAIL}"
```

Comment:
worth moving this to constants?  could go either way here tbh

Reply:
yeah i think it makes more sense in constants too